doc/22_Administration_of_Pimcore/02_Cleanup_Data_Storage.md

```diff
@@ -63,6 +63,13 @@ Deleting items in Pimcore moves them to the recycle bin first. The recycle bin w
 so the references are kept in the database but the contents itself are dumped into files in `var/recyclebin/`.   
 You can review items in the bin in the admin user-interface under *Tools* > *Recycle Bin*, there it's also possible to 
 flush the entire contents. 
+
+It's also possible to execute following command using cron or maintenance task: 
+```bash
+./bin/console  pimcore:recyclebin:cleanup --older-than-days=60
+```
+
+Or such extension: https://pimcore.com/en/developers/marketplace/blackbit_digital_commerce/recycle-bin-cleanup_e292447
   
```

Comment:
```suggestion
```
I love BlackBit, but the repository looks empty and does nothing at the moment, maybe it's a facade public repo for a enterprise one